single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -1021,20 +1021,32 @@ protected void insertArrowsAndLabels(String prefixId,
                                          GraphMetadata metadata,
                                          DiagramLabelProvider initProvider,
                                          DiagramStyleProvider styleProvider) {
+        if (!(n instanceof FeederNode)) {
+            throw new AssertionError("Node n must be a feeder node");
+        }
+
+        FeederNode feederNode = (FeederNode) n;
         InitialValue init = initProvider.getInitialValue(n);
 
+        Optional<String> label1 = feederNode.getDirection() == BusCell.Direction.TOP ? init.getLabel1() : init.getLabel2();
```

Comment:
I'm wondering if this should always be done like that. It's maybe better to have that as a default behaviour, but which could be changed through LayoutParameters? Indeed, some users seem to worry a lot about symmetry!

Reply:
'feederArrowSymmetry' parameter added